gempy/core/model.py

```diff
@@ -1355,4 +1357,24 @@ def set_gravity_interpolator(self, density_block=None,
         if compile_theano is True:
             self.interpolator_gravity.compile_th_fn_grav(density_block, pos_density, inplace=True)
 
-        return self.additional_data.options
\ No newline at end of file
+        return self.additional_data.options
+
+    def set_crs(self, crs_code: str):
```

Comment:
@flohorovicic I'm not quite sure what the use for setter and getter functions is when used in this way. With this implementation I can also just get the crs by `obj.crs` and override it anytime with `obj.crs = something`, as its a public attribute. Also it really simply sets and gets what the user wants, no matter if its a valid crs 🤔 

If you want to have proper getter and setters of a hidden attribute, I'd recommend making use of Python's `property` decorator:
```python
class Class:
    def __init__(self, crs):
        self._crs = crs

    @property
    def crs(self):
        return self._crs

    @crs.setter
    def crs(self, value):
        # maybe additional code for checking if its a proper crs the user is trying to set
        self._crs = value
```

Otherwise why not just go with
```python
class Class:
    def __init__(self, crs):
        self.crs = crs
```

Reply:
Hi @pytzcarraldo,

Thanks for the note - actually, already planned to do that and it is now implemented (even before I saw your comment :-D ), but I used a double '__' - any recommendation on that? I followed https://www.python-course.eu/python3_properties.php .

I also added a test. The reason for using getters and setters in the previous version (the way I used in pynoddy, so just what I was used to...) was to enable tests (e.g. if a crs that is set actually exists) and call methods (e.g. to transform data automatically when crs is changed). But for sure, the right way is to use the @property decorator, I quite like the functionality.

Reply:
Yeah, apparently dunderscores are the convention for property - didn't know that, thanks! 😄